packages/runtime-common/realm-index-query-engine.ts

```diff
@@ -152,6 +153,7 @@ export class RealmIndexQueryEngine {
           errorDetail: instance.error,
           scopedCssUrls,
           lastKnownGoodHtml: instance.isolatedHtml ?? null,
+          cardTitle: instance.searchDoc?.title ?? null,
```

Comment:
I'm adding this so that we can show the card's title in the delete modal:

<img width="498" alt="image" src="https://github.com/user-attachments/assets/52f3f5f9-60a9-4132-a247-0e105bf89ef8" />